adhocracy4/follows/static/follows/FollowButton.jsx

```diff
@@ -39,10 +49,29 @@ class FollowButton extends React.Component {
   render () {
     return (
       <span className="btngroup btngroup-gray">
-        <button className="btn btn-sm btn-dark btn-primary" type="button" onClick={this.toggleFollow.bind(this)}>
-          <i className={this.state.followed ? 'fa fa-star' : 'fa fa-star-o'} aria-hidden="true" />
-          &nbsp;{this.state.followed ? django.gettext('Unfollow') : django.gettext('Follow')}
-        </button>
+        <span className="dropdown">
+          <button className="btn btn-secondary dropdown-toggle" type="button" id="follow-dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <i className="fa fa-star" aria-hidden="true" />&nbsp;
+            {this.state.followed ? django.gettext('Unfollow') : django.gettext('Follow')}&nbsp;
+            <i className="fa fa-caret-down" aria-hidden="true" />
+          </button>
+          <span className="dropdown-menu" aria-labelledby="follow-dropdown">
+            <button className="dropdown-item select-item" onClick={this.disableFollow.bind(this)}>
```

Comment:
Is the class name `select-item` not a bit over generic?

Reply:
I had something like `follow`before which I found too specific. You have any suggestions for something in between?